node/tessel-export.js

```diff
@@ -1399,7 +1418,25 @@ function getWifiInfo() {
                   if (error) {
                     reject(error);
                   } else {
-                    network.ips = ipResults.split('\n');
+                    network.ip = ipResults
+                      .split('\n') // split multiple line string into array of strings
+                      .filter((string) => string.includes('Bcast'))[0] // find the string containing a reference to "Bcast" (Broadcast IP)
+                      .trim() // remove extra whitespace from either side of the string
+                      .split(' ') // split the string by whitespace between words
+                      .filter((string) => string.includes('Bcast'))[0] // find the string containing a reference to "Bcast"
+                      .split(':')[1]; // split the string by the ":", i.e. "Bcast:10.0.0.11" and grab the number as the 2nd item in that array
```

Comment:
I agree, however because it's in the `try...catch` block, an error will be caught and known, as well as when testing on multiple networks and security types, this hasn't failed (yet).

I'm definitely open to a more efficient solution with less assumptions. My Regex abilities are definitely weak in situations like this.